packages/grafana-toolkit/src/cli/utils/useSpinner.ts

```diff
@@ -1,8 +1,8 @@
 import ora = require('ora');
 
-type FnToSpin<T> = (options: T) => Promise<void>;
+type FnToSpin<T> = (options: T) => Promise<any>;
 
-export const useSpinner = <T = any>(spinnerLabel: string, fn: FnToSpin<T>, killProcess = true) => {
+export function useSpinner<T = void>(spinnerLabel: string, fn: FnToSpin<T>, killProcess = true) {
```

Comment:
IIUC, this is the only real change -- the rest just come from it?  In general, it seems better to export functions rather than constant implementations of a function!

:+1:

Reply:
The timing change at https://github.com/grafana/grafana/pull/27868/files#diff-6e99f8c06e5db7f6d569cb8aec3bc74bR66 could be something more real as well

Reply:
great -- for PRs like this where there are only a couple lines with meaningful changes, can you point them out in the PR with a comment?

Reply:
#27906